### PR TITLE
specviz2d: expose specviz helper

### PR DESCRIPTION
**Pull request by kecnry**
### Description

This pull request exposes `specviz2d.specviz` to match `cubeviz.specviz` and `mosviz.specviz`.

Needed for upcoming specviz2d docs PR (docs not included as part of this PR since the docs for all of specviz need to be populated).

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [x] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [x] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Did the CI pass? If not, are the failures related?
- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [x] Is a milestone set?
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: jdaviz/configs/specviz2d/helper.py [hunk 1]
```diff
@@ -22,6 +23,16 @@ def __init__(self, *args, **kwargs):
         spec2d = self.app.get_viewer("spectrum-2d-viewer")
         spec2d.scales['x'].observe(self._update_spec1d_x_axis)
 
+    @property
+    def specviz(self):
+        """
+        A specviz helper (`~jdaviz.configs.specviz.helper.Specviz`) for the Jdaviz
+        application that is wrapped by mosviz
```
Comment (ojustino) [inline jdaviz/configs/specviz2d/helper.py:30]:
```suggestion
        application that is wrapped by specviz2d
```

**Review by ojustino — APPROVED** [review]
I approve contingent on a small change to the docstring.

File: jdaviz/configs/specviz2d/helper.py (hunk 1 above)
Comment (kecnry) [reply jdaviz/configs/specviz2d/helper.py]: woops - thanks!  Fixed

File: CHANGES.rst [hunk 2]
```diff
@@ -42,6 +42,8 @@ Specviz2d
 
 - Spectral extraction plugin. [#1514, #1560, #1562]
 
+- Expose ``specviz2d.specviz`` helper. [#1577]
```
Comment (pllim) [inline CHANGES.rst:45]:
```suggestion
- New ``specviz2d.specviz`` helper property to directly access Specviz functionality from Specviz2d. [#1577]
```

File: jdaviz/configs/specviz2d/helper.py [hunk 3]
```diff
@@ -22,6 +23,16 @@ def __init__(self, *args, **kwargs):
         spec2d = self.app.get_viewer("spectrum-2d-viewer")
         spec2d.scales['x'].observe(self._update_spec1d_x_axis)
 
+    @property
+    def specviz(self):
+        """
+        A specviz helper (`~jdaviz.configs.specviz.helper.Specviz`) for the Jdaviz
+        application that is wrapped by specviz2d
```
Comment (pllim) [inline jdaviz/configs/specviz2d/helper.py:30]:
```suggestion
        A Specviz helper (`~jdaviz.configs.specviz.helper.Specviz`) for the Jdaviz
        application that is wrapped by Specviz2d.
```

**Review by pllim — COMMENTED** [review]
Should this be tested in some way? Or maybe a matter of modifying an existing test to use this new property?

**Comment by pllim** [comment]
User doc should also be updated but maybe that can wait until #1580 .

File: jdaviz/configs/specviz2d/helper.py (hunk 3 above)
Comment (kecnry) [reply jdaviz/configs/specviz2d/helper.py:30]: Ok, I'll make these changes across all three instances then (cubeviz and mosviz as well).

**Comment by kecnry** [comment]
> Should this be tested in some way? Or maybe a matter of modifying an existing test to use this new property?

Probably, but I'm not sure we have enough existing tests in specviz2d at all.  I could just access the property and do an isinstance check?

> User doc should also be updated but maybe that can wait until https://github.com/spacetelescope/jdaviz/pull/1580 .

Yes, I needed this for the docs but didn't want to sneak code changes into a docs PR, so docs updates are intentionally missing here.

**Comment by pllim** [comment]
> do an isinstance check?

Yes, that should be good enough for now. Thanks!

**Pull request merged**